### PR TITLE
Update lastfmplus to use Last.fm API v2.0

### PR DESCRIPTION
**Pull request by pdf**
I've created an API key and hard-coded it here, though ideally the key would be provided by someone affiliated with the MetaBrainz project.  Alternatively, it may be possible to add a configuration option and require all users to sign up for their own key, but that seems sub-optimal from a usability viewpoint, and updating the config dialog is made more painful by the lack of the `options_lastfmplus.ui` file in the repo.

### CODE REVIEW

**Comment by monkiibiz** [comment]
When running this, the web service returns a 404. When looking in the logs, it appears the `?method` in the beginning of the string gets encoded to `%3Fmethod` before the service call is executed:
`E: 12:07:59 Network request error for http://ws.audioscrobbler.com:80/2.0/?method=track.gettoptags&artist=MIKA&track=Grace%20Kelly&mbid=e729519d-8780-35cd-a354-ba47e10807f6&api_key=8b331b492e4afd88e8ef9c9aa872c785: Error downloading http://ws.audioscrobbler.com:80/2.0/%3Fmethod=track.gettoptags&artist=MIKA&track=Grace%2520Kelly&mbid=e729519d-8780-35cd-a354-ba47e10807f6&api_key=8b331b492e4afd88e8ef9c9aa872c785 - server replied: Not Found (QT code 203, HTTP code 404)`

**Comment by pdf** [comment]
@monkiibiz I don't see that, can you tell me what software versions you're using?

**Comment by monkiibiz** [comment]
Running the latest daily build
1.4.0dev5_win_20160622100106

**Comment by zas** [comment]
@pdf @monkiibiz : any update on the 404 issue ?

**Comment by pdf** [comment]
I can't reproduce it...

File: plugins/lastfmplus/__init__.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 LASTFM_HOST = "ws.audioscrobbler.com"
 LASTFM_PORT = 80
+LASTFM_API_KEY = "8b331b492e4afd88e8ef9c9aa872c785"
```
Comment (zas) [inline plugins/lastfmplus/__init__.py]: This API key is supposed to be changed by plugin user ?

Comment (pdf) [inline plugins/lastfmplus/__init__.py]: Please see the original post for this issue.

**Comment by Holzhaus** [comment]
Can this be merged? Seems like the problem by @monkiibiz was caused by temporary client-
 or server-side connection issues. It works fine for me, even when copy&pasting the URL from the error message manually into my browser.

**Comment by Holzhaus** [comment]
Any updates on this?

**Comment by samj1912** [comment]
Can you please rebase this against 2.0 and 1.0? (Whatever API version you are aiming at)
API 2.0 is strictly py3/qt5

**Comment by pdf** [comment]
I don't normally write Python, so I'm not sure what's involved there, and I think there are some outstanding questions around how API keys should be handled.  I'm happy for someone else to take over this PR.

**Pull request closed without merging**